api/server/sdk/server.go

```diff
@@ -107,6 +107,10 @@ type ServerConfig struct {
 	StoragePolicy policy.PolicyManager
 	// Security configuration
 	Security *SecurityConfig
+	// ServerExtensions allow extending the SDK gRPC server
+	// with callback functions that are executed in order
+	// at the end of Server.Start()
```

Comment:
Thanks. I think you should add a small comment to showing an example of how it can be used. Then merge it